tests/arrays_test_generated.h

```diff
@@ -140,22 +149,34 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(8) ArrayStruct FLATBUFFERS_FINAL_CLASS {
   static const flatbuffers::TypeTable *MiniReflectTypeTable() {
     return ArrayStructTypeTable();
   }
-  ArrayStruct() {
-    memset(static_cast<void *>(this), 0, sizeof(ArrayStruct));
+  ArrayStruct()
+      : a_(0),
+        b_(),
+        c_(0),
+        padding0__(0),
+        padding1__(0),
+        padding2__(0),
+        d_(),
+        e_(0),
+        padding3__(0),
+        f_()
+  {
+    (void)padding0__;
+    (void)padding1__;
+    (void)padding2__;
+    (void)padding3__;
   }
   ArrayStruct(float _a, int8_t _c, int32_t _e)
       : a_(flatbuffers::EndianScalar(_a)),
+        b_(),
         c_(flatbuffers::EndianScalar(_c)),
         padding0__(0),
         padding1__(0),
         padding2__(0),
+        d_(),
```

Comment:
found [this](https://stackoverflow.com/a/7760316) SO answer, which references to standard
all its references can be found in each standard draft since c++11 found [here](https://en.cppreference.com/w/cpp/links) - 11, 14, 17 and 20 (under different numbers, but their texts are the same)

Reply:
Thanks for looking this up.
Adding a test for this is maybe overkill, but if you wanted, you could use placement-new on memory that was initialized to not-0 to test that the constructor does its job?

Reply:
sure
https://github.com/google/flatbuffers/pull/5938/commits/88cdd6ffcbd67d3db2e7fb2090e5bd9b6665eed6